core/templates/pages/exploration-editor-page/services/graph-data.service.ts

```diff
@@ -53,6 +43,16 @@ angular.module('oppia').factory('GraphDataService', [
       recompute: function() {
         _recomputeGraphData();
       },
+      /**
+       * @return graphData - Directed graph visualization input. This object
+       * includes the following keys:
+       * - nodes: Objects with keys of nodeids and values of node names.
+       * - links: list of objects. Each object represents a directed link
+       * between two nodes, and has keys 'source' and 'target', the values
+       * of which are the names of the corresponding nodes.
+       * - initStateName: the name of the initial state.
+       * - finalStateName: the name of the final state.
+       */
       getGraphData: function() {
```

Comment:
Does this function also need a JSDoc?

Reply:
I think in this particular file, online this function we have to document. No other actual returns can be found. If I am wrong, please guide me through.

Reply:
oops, sorry! I thought the comment was placed incorrectly. Thanks for clarifying.